documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
+
+The Lua Users Foundation (the Foundation) is an association of individuals in a modular umbrella structure consisted of several workgroups, detailed in the Lua Users Foundation Constitution.
+The Foundation formal decision-making process is governed by the Lua Users Foundation Constitution, according to the Foundation Goals and Values, towards our Mission.
+
+## The Mission
+
+  * Support and promote the Lua programming language, its community and ecosystems.
+    
+## The Values
+
+  * The Lua programming language is developed by the Lua Team and defined by the Lua Programming Manual. Furthermore:
+  * The Lua programming language is small, fast, easy to learn and use. 
+  * The Lua programming language philosophy is less is more.
+  * The Lua ecosystem is diverse, it includes many libraries, frameworks, implementations, dialects and related languages.
+  * The Lua community is inclusive. We welcome and encourage participation by everyone. We are committed to foster a diverse community with respect of personal characteristics such as, but not limited to, gender identity or expression, sexual orientation, ethnicity, age, religion, disability, class, geographic location, background and experience.
+  * Lua and related technologies should have wider adoption in the world.
+  * The Lua Users Foundation is a transparent and not for profit endeavour. 
+  * The Lua Users Foundation values simplicity and accessibility (easy to learn and use) in our work.
+
+## The Goals
+
+ * To aid the Lua Team and the Lua programming language implementers.
+ * To support and foster critical projects such as Lua libraries (modules) and frameworks ecosystems.
```

Comment:
s/frameworks ecosystems/framework ecosystems/

... I think. :)

Reply:
I think frameworks ecosystems is the correct! :)

Reply:
I think catwell is correct here

Reply:
`framework ecosystems` or `ecosystems of frameworks` (less wieldy and with the same meaning).